index.html

```diff
@@ -9,10 +9,850 @@
+                                  <span class="loginwith">Sign in with<br />social network</span>
+                                  
+                                  <button class="social-signin facebook">Log in with facebook</button>
+                                  <button class="social-signin twitter">Log in with Twitter</button>
+                                  <button class="social-signin google">Log in with Google+</button>
+                                </div>
+                                <div class="or">OR</div>
+                              </div>
+                            </div>
+                
+           </div>
+       </div>
+       <div class="modal-bg-cart">
+            <div class="modal-content-cart">
+                <div class="cart-header">
+               <h3 class="cart-header-o">Cart overview</h3> <span class="close-cart" >&times;</span></div>
+                   <div class="cart-items">
+                       
+                   </div>   
+                   <div class="cart-footer">
+                      <div class="cart-total"> <p>Total</p> <p class="totall">$0</p></div>
+                    <div >
+                        <button type="submit" class="cart-submit" > Checkout</button>
+                    </div>
+                   </div>
+                 
+            </div>
+        </div>
+        <section >
+            <div>
+               
+
+                <!-- carousel -->
+                <!-- Slideshow container -->
+<div class="slideshow-container">
+
+        <!-- Full-width images with number and caption text -->
+        <div class="mySlides fade">
+          <div class="numbertext">1 / 3</div>
+          <img src="img1.jpg" style="width:100%">
+          <div class="text">Caption Text</div>
+        </div>
+      
+        <div class="mySlides fade">
+          <div class="numbertext">2 / 3</div>
+          <img src="img2.jpg" style="width:100%">
+          <div class="text">Caption Two</div>
+        </div>
+      
+        <div class="mySlides fade">
+          <div class="numbertext">3 / 3</div>
+          <img src="img3.jpg" style="width:100%">
+          <div class="text">Caption Three</div>
+        </div>
+      
+        <!-- Next and previous buttons -->
+        <a class="prev" onclick="plusSlides(-1)">&#10094;</a>
+        <a class="next" onclick="plusSlides(1)">&#10095;</a>
+      </div>
+      <br>
+      
+      <!-- The dots/circles -->
+      <div style="text-align:center">
+        <span class="dot" onclick="currentSlide(1)"></span>
+        <span class="dot" onclick="currentSlide(2)"></span>
+        <span class="dot" onclick="currentSlide(3)"></span>
+      </div>
+     <!--end  -->
+                
+                <div>
+
+                </div>
+                <!-- /cardds -->
+                <div class="cardds-section" id="computers">
+                    <p class="cardd_title">Computers and Accessories</p>
+
+                    <div class="cardds">
```

Comment:
Fix typos

Reply:
they aren't really typos... i used 'cardd' and not 'card' because earlier i wanted to use bootstrap's caurosel and bootstrap had a default "card" class and that messed up with my entire work.... same with the "searchh" so i just included a last letter for those classes